views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -71,10 +71,11 @@ define([
         }
 
 
-        var patternMask = interaction.attr('patternMask');
-        var expectedLength = parseInt(interaction.attr('expectedLength'), 10);
-        var maxWords = parsePattern(patternMask,'words');
-        var maxChars = parsePattern(patternMask,'chars');
+        var patternMask = interaction.attr('patternMask'),
+        expectedLength = parseInt(interaction.attr('expectedLength'), 10),
+        expectedLines = parseInt(interaction.attr('expectedLines'),10),
+        maxWords = parsePattern(patternMask,'words'),
+        maxChars = parsePattern(patternMask,'chars');
```

Comment:
It's not supposed to, It's just suppose to return "there's nothing i can match here" with the `null` of something like that.